scandium-core/src/main/java/org/eclipse/californium/scandium/config/DtlsConnectorConfig.java

```diff
@@ -1107,6 +1143,9 @@ public DtlsConnectorConfig build() {
 			if (config.sniEnabled == null) {
 				config.sniEnabled = Boolean.TRUE;
 			}
+			if (config.verifyRequestOnResumptionEnabled == null) {
+				config.verifyRequestOnResumptionEnabled = Boolean.FALSE;
```

Comment:
Without surprise, I would prefer to activate it by default. :grin: 

Some arguments :
1. The "more secure way" activated by default forces users to think about potential issues before to relax it.
2. We don't change behavior regarding previous version of scandium by default.

If you think I'm wrong, just let it like this. This is not a big deal :)

Reply:
The behaviour implemented by this PR matches RFC6347, page 21, figure 2.
Therefore I think, this is the right default. If you prefer, just change it for leshan and we will see, if the "more security" shows up some how.

Reply:
I update the implementation of `isClientInControlOfSourceIpAddress` to take more cases into account.
Currently I think, the right way to do this, is to flexible adapt the mechanism to the situation. Counting successful and stale VERIFY_REQUESTs and stale "resumption handshakes" should provide a dynamic switch, if a  VERIFY_REQUEST should be used or not. I hope, I can provide such a solution before the next Milestone release, but not now. So I would postpone the default decision to that timepoint and leave it for now as `config.verifyRequestOnResumptionEnabled = Boolean.FALSE`.

Reply:
> Currently I think, the right way to do this, is to flexible adapt the mechanism to the situation. 

I have difficulty to imagine that, but I'm curious, I keep tuned :D